### PR TITLE
ENH: Add a Jupyter notebook on ET's data QC

### PR DESCRIPTION
**Pull request by oesteban**

### CODE REVIEW

File: code/eyetracking/schedule.tsv [hunk 1]
```diff
@@ -1,5 +1,5 @@
-session	day	PE	fixation_edf	qct_edf	rest_edf	bht_edf
-1	2023-10-20	LR	fixation_2023-10-20_18h48.03.561_5_session_1.EDF	qct_2023-10-20_19h40.38.964_2_session_1.EDF	rest_2023-10-20_19h48.22.344_0_session_1.EDF	bht_2023-10-20_20h10.50.517_0_session_1.EDF
+session	day	PE	dwi_edf	qct_edf	rest_edf	bht_edf
```
Comment (oesteban) [inline code/eyetracking/schedule.tsv]: why this change?

File: code/eyetracking/schedule.tsv [hunk 2]
```diff
@@ -1,5 +1,5 @@
...
+1	2023-10-20	LR	fixation_2023-10-20_18h46.06.106_4_session_1.EDF	qct_2023-10-20_19h40.38.964_2_session_1.EDF	rest_2023-10-20_19h48.22.344_0_session_1.EDF	bht_2023-10-20_20h10.50.517_0_session_1.EDF
```
Comment (oesteban) [inline code/eyetracking/schedule.tsv]: Are you sure this is correct? The old one looks better (2 minutes later)

File: code/eyetracking/edf-to-bids.ipynb
Comment (oesteban) [inline code/eyetracking/edf-to-bids.ipynb:1]: Why is this notebook back? I haven't checked but looks like an accident, correct?

File: code/eyetracking/ETdata_extraction.py
Comment (oesteban) [inline code/eyetracking/ETdata_extraction.py:1]: This looks like an accidental addition, correct? It didn't look like we have anything here that is not in the `eyetrackingrun.py` module.

File: code/eyetracking/ETdata_plotting.py
Comment (oesteban) [inline code/eyetracking/ETdata_plotting.py:1]: Also looks like an accidental addition.

**Review by oesteban — COMMENTED** [review]
Just a very fast pass - can you confirm whether the files I marked were added accidentally?

File: code/eyetracking/edf-to-bids.ipynb
Comment (esavary) [reply code/eyetracking/edf-to-bids.ipynb:1]: Yes, they were added by accident (same for the other two files). I've removed them now.

File: code/eyetracking/schedule.tsv (hunk 1 above)
Comment (esavary) [reply code/eyetracking/schedule.tsv]: It was an error (same for the comment below). I reverted the schedule to the correct version.

File: docs/data-management/edf-to-bids.ipynb
Comment (oesteban) [inline docs/data-management/edf-to-bids.ipynb:1]: This file is also an oversight, when present, `mkdocs serve` fails.

File: mkdocs.yml [hunk 3]
```diff
@@ -38,8 +38,7 @@ nav:
       - Notes on software installation and tips: data-collection/notes-software.md
       - Legacy procedure for the ET mirror: data-collection/legacy-et.md
       - Converting physio into BIDS (notebook): data-management/physio-to-bids.ipynb
-      - Converting eye-tracking into BIDS: data-management/edf-to-bids.md
-      - QA/QC criteria: data-management/qaqc-criteria.md
+      - Converting eye-tracking into BIDS (notebook): data-management/edf-to-bids.ipynb
```
Comment (oesteban) [inline mkdocs.yml]: ??

File: code/eyetracking/visualizing-and-qc-eyetrack-data.ipynb [hunk 4]
```diff
@@ -0,0 +1,1153 @@
...
+    "        if stop_time is not None:\n",
+    "\n",
+    "            subtask_data = et_qct.recording[\n",
+    "                (et_qct.recording[\"timestamp\"] >= start_time)\n",
+    "                & (et_qct.recording[\"timestamp\"] <= stop_time)\n",
+    "            ]\n",
+    "\n",
+    "            df_name = f\"df_{subtask_type}_{n_task}\"\n",
+    "            subtask_dataframes[df_name] = subtask_data\n"
+   ]
```
Comment (oesteban) [inline code/eyetracking/visualizing-and-qc-eyetrack-data.ipynb]: why not loading into a pandas dataframe and avoid all the finicky parsing?

File: code/eyetracking/visualizing-and-qc-eyetrack-data.ipynb [hunk 5]
```diff
@@ -0,0 +1,1153 @@
...
+    "    )\n",
+    "plt.axvline(\n",
+    "    x=timestamps_in_seconds[\"start_bh_block\"] + 40,\n",
+    "    color=\"violet\",\n",
+    "    linestyle=\"--\",\n",
+    "    label=\"break\",\n",
+    ")\n",
+    "for n in np.arange(1, 5):\n",
+    "    plt.axvline(\n",
+    "        x=timestamps_in_seconds[\"start_bh_block\"] + 40 * (n + 1) + 10 * n,\n",
```
Comment (oesteban) [inline code/eyetracking/visualizing-and-qc-eyetrack-data.ipynb]: magic numbers, please don't force the reader to understand on their own
